v2/thirdparty/common-customizations/saml/with-boxyhq/integration-steps.mdx

```diff
@@ -233,6 +233,21 @@ curl --location --request PUT '^{coreInjector_uri_without_quotes}/recipe/multite
 
 ## 6) Configure the SAML provider for the tenant
 
+To configure SAML login with SuperTokens, ensure that you use the correct provider name in the third-party config.
+Below is a list of supported SAML SSO provider names.
+
+- microsoft entra id <br/>
+- microsoft ad fs <br/>
+- okta <br/>
+- auth0 <br/>
+- google <br/>
+- onelogin <br/>
+- pingone <br/>
+- jumpcloud <br/>
+- rippling <br/>
+- openid <br/>
+- generic saml <br/>
+
```

Comment:
do these provider names have to be all small caps?

Reply:
No, it is case-insensitive